packages/crawler/tests/e2e/specs/index.e2e.ts-snapshots/should-generate-raw-data-correctly-for-1-version-and-2-prefixes-1-chrome-darwin.txt

```diff
@@ -0,0 +1 @@
+{"latest":{"my-prefix":[{"p-text":{"properties":{"tag":"p","size":"small","weight":"regular","align":"start","color":"primary","ellipsis":false,"theme":"light"},"children":"First prefix","hostPdsComponent":null}}],"":[{"p-text":{"properties":{"tag":"p","size":"small","weight":"regular","align":"start","color":"primary","ellipsis":false,"theme":"light"},"children":"Second prefix","hostPdsComponent":null}}]}}
```

Comment:
I don't think this works in the CI, since the snapshots are generated by your local environment macOS. … see `darwin` in the snapshot name.